drivers/biamp/ntp.cr

```diff
@@ -0,0 +1,81 @@
+# Biamp ATP/NTP protocol utilities.
+# https://support.biamp.com/Audia-Nexia/Control/Audia-Nexia_Text_Protocol
+module Biamp::NTP
+  record Command,
+    type : Type,
+    device : Int32,
+    attribute : String,
+    instance : Int32? = nil,
+    index_1 : Int32? = nil,
+    index_2 : Int32? = nil,
+    value : String | Int32 | Float32 | Nil = nil do
+    def self.[](type : Type, device, attribute)
```

Comment:
Less code here, but more room for error in usage. The square bracket syntax this provides lets a `Command` be declared as it would appear, but with each field type checked and captured into this struct correctly. Using `.new` forces `value` to always be specified as a named arg as preceding fields are optional (at the NTP protocol level).

There was an error in this original implementation though (thank you for prodding!), where `value` could be incorrectly captured into the `instance` attribute. Fixed in cd97b52 and also condensed a little.